src/cmd/tools/m3ctl/yaml/peeker.go

```diff
@@ -48,6 +49,8 @@ func peeker(data []byte) (string, proto.Message, error) {
 
 	// now the payload is of known type
 	// unmarshal it and return the proto.Message
+	placementPath := placements.DefaultPath + peek.Service + "/placement"
```

Comment:
If this is run with a config that has not been updated, `placementPath` will be `/api/v1/services//placement`.
Should we maybe default `peek.Service` to `m3db` if it is not set?

Reply:
Good point. PR has been updated.